### PR TITLE
1.3

### PR DESCRIPTION
**Pull request by redxblock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # LastStand
 LastStand 1.7.10's last modpack.
+<<<<<<< HEAD
```
Comment (fractalis) [inline README.md]: Minor; probably don't want the git-diff syntax in here.

File: client/curse/LastStand/manifest.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
   "manifestType": "minecraftModpack",
   "manifestVersion": 1,
   "name": "LastStand",
-  "version": "1.2",
+  "version": "1.0",
```
Comment (fractalis) [inline client/curse/LastStand/manifest.json]: Should this be 1.3?

File: client/curse/LastStand/overrides/config/RunicDungeons/ChestGeneration.cfg [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+        IC2:itemIngot:0:2:5:0
+        IC2:itemIngot:1:2:5:50
+        IC2:itemOreIridium:0:1:2:15
+        IC2:itemToolBronzePickaxe:0:1:1:3
+        IC2:itemToolBronzeSword:0:1:1:3
+        IC2:itemArmorBronzeHelmet:0:1:1:3
+        IC2:itemArmorBronzeChestplate:0:1:1:3
+        IC2:itemArmorBronzeLegs:0:1:1:3
+        IC2:itemArmorBronzeBoots:0:1:1:3
+        PortalGun:PortalGunBlue:0:1:1:2
```
Comment (fractalis) [inline client/curse/LastStand/overrides/config/RunicDungeons/ChestGeneration.cfg]: Not sure if you intend on allowing Portal Guns in final release given general issues with (chunkloading?), but throwing out a consideration to remove them in the RD Chest Gen

File: client/curse/LastStand/overrides/scripts/RunicDungeons.zs [hunk 4]
```diff
@@ -0,0 +1,2 @@
+//add shaped recipes
+recipes.addShaped(<runicdungeons:item.magicchalk>, [[<academy-craft:ac_Item_matrixCore:2>, <Thaumcraft:blockStoneDevice:2>, <academy-craft:ac_Item_matrixCore:2>], [<Thaumcraft:blockStoneDevice:2>, <witchery:chalkritual>, <Thaumcraft:blockStoneDevice:2>], [<academy-craft:ac_Item_matrixCore:2>, <Thaumcraft:blockStoneDevice:2>, <academy-craft:ac_Item_matrixCore:2>.noReturn()]]);
```
Comment (fractalis) [inline client/curse/LastStand/overrides/scripts/RunicDungeons.zs]: Awesome! :+1:

**Comment by redxblock** [comment]
completed

File: README.md (hunk 1 above)
Comment (redxblock) [inline README.md]: Fixed. github for windows is garbage when you are tired..

File: client/curse/LastStand/manifest.json (hunk 2 above)
Comment (redxblock) [inline client/curse/LastStand/manifest.json]: fixed. 7069b85

File: client/curse/LastStand/overrides/config/RunicDungeons/ChestGeneration.cfg (hunk 3 above)
Comment (redxblock) [inline client/curse/LastStand/overrides/config/RunicDungeons/ChestGeneration.cfg]: I had thought to keep it in and remove it from the dungeon loot from the next branch 1.4. Let's talk to dingo about removing them altogether i guess. I haven't gotten to play the pack enough to enjoy them yet. LOL.

**Pull request merged**